canonical-websites/www.ubuntu.com
# 2650: Build beta.ubuntu.com primary nav

## Done

- Built primary nav according to [copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y) and [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2644)
- Built download tab according to [copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y) and [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2647)
- Built the remaining navs to the latest design and copy (**NOT finalised - these will be properly built in a separate PR**):
  - Secondary nav ([copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y), [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2648))
  - Global nav ([copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y), [design](https://github.com/canonical-websites/www.ubuntu.com/issues/2643))

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Play around with it a bit and check that the primary, secondary and global nav all work
- Check that primary nav's copy and design match the [copydoc](https://docs.google.com/document/d/14EJRVoh2Hfc_K3ZFZbPPRkqeZ_NQUwDJp9vnyIuadf4/edit#heading=h.g1x8grgh273y) and [signed-off design](https://github.com/canonical-websites/www.ubuntu.com/issues/2644)

## TODO

- Make primary nav stick to the top of the page (except when at the top, because that's where the global nav lives)
- Add animation to dropdowns
- Any semblance of responsive design

## Issues

Fixes #2650, fixes #2652
